Model/Menu.php

```diff
@@ -45,4 +45,151 @@ public function saveStores(array $stores)
         }
         $connection->commit();
     }
+
+    /**
+     * Gets menu id.
+     *
+     * @return int|null
+     */
```

Comment:
In modules, I think it's better to use:

```php
/**
 * @inheritdoc
 */
```

so in case something changes in the interface we don't need to modify here too

Reply:
updated